src/parser/utils.ts

```diff
@@ -58,7 +58,7 @@ export function haveBuffer(): boolean {
 
 /** Callable in any environment to check if value is a Buffer */
 export function isBuffer(value: unknown): value is Buffer {
-  return typeof value === 'object' && value?.constructor?.name === 'Buffer';
+  return typeof value === 'object' && value?.constructor === Buffer;
```

Comment:
How about e.g.

```suggestion
  return isUint8Array(value) && typeof value?.toJSON === 'function';
```

Reply:
I think this would work! Thanks for the suggestion, I'll merge it and see if I can safely minify it on my end